### PR TITLE
spelling correction

### PR DESCRIPTION
**Pull request by shefali12-ab**
I have corrected the spelling of some of the .md files.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -119,14 +119,14 @@ version 2.1, available at
...
 For answers to common questions about this code of conduct, see the FAQ at
-[https://www.contributor-covenant.org/faq][FAQ]. Translations are available
+[https://www.contributor-covenant.org/faq][faq]. Translations are available
 at [https://www.contributor-covenant.org/translations][translations].
 
 [homepage]: https://www.contributor-covenant.org
 [v2.1]: https://www.contributor-covenant.org/version/2/1/code_of_conduct.html
-[Mozilla CoC]: https://github.com/mozilla/diversity
-[FAQ]: https://www.contributor-covenant.org/faq
+[mozilla coc]: https://github.com/mozilla/diversity
```
Comment (AbhijithGanesh) [inline CODE_OF_CONDUCT.md:130]: It'd be better if these words are capitalized the same way.

File: resources/Introduction-to-Cloud-Native.md [hunk 2]
```diff
@@ -1,48 +1,51 @@
-# Introduction to Cloud Native #
+# Introduction to Cloud Native
 
-### **What is Cloud Native?**  <hr/>
+### **What is Cloud Native?** <hr/>
```
Comment (AbhijithGanesh) [inline resources/Introduction-to-Cloud-Native.md:3]: The `<hr/>` is not required.

File: resources/Introduction-to-Cloud-Native.md [hunk 3]
```diff
@@ -1,48 +1,51 @@
...
 
-***The Cloud Native Computing Foundation(CNCF)***  defines Cloud Native as technologies that empower organizations to build and run scalable applications in modern, dynamic environments 
+**_The Cloud Native Computing Foundation(CNCF)_** defines Cloud Native as technologies that empower organizations to build and run scalable applications in modern, dynamic environments
```
Comment (AbhijithGanesh) [inline resources/Introduction-to-Cloud-Native.md:5]: *** and **_ in symbol does the same thing.

File: resources/Introduction-to-Cloud-Native.md [hunk 4]
```diff
@@ -1,48 +1,51 @@
...
 such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach.
 
-**So simply defined, cloud-native is building an application for the cloud by making efficient use of the resources the cloud provides.** 
+**So simply defined, cloud-native is building an application for the cloud by making efficient use of the resources the cloud provides.**
 
-### **Need for cloud-native**  <hr/>
+### **Need for cloud-native** <hr/>
```
Comment (AbhijithGanesh) [inline resources/Introduction-to-Cloud-Native.md:10]: Again, `<hr/>` is not required.

File: resources/cloud_native_architecture.md [hunk 5]
```diff
@@ -1,30 +1,32 @@
...
+The following is the summary :
 
+- **_Design for automation_** : Automated processes can repair, scale, deploy your system far faster than people can. <br/><br/>
+- **_Be smart with state_** : Storing of 'state', be that user data or system state is the hardest aspect of architecting a distributed, cloud-native architecture. You should therefore architect your system to be intentional about when, and how, you store state.</br></br>
+- **_Favour managed services_** : Cloud is more than just infrastructure. Most cloud providers offer a rich set of managed services, providing all sorts of functionality that relieve you of the headache of managing the backend software or infrastructure.
 
-Now that we know about the principles of cloud-native architecture, we now move to discuss a bit more about application archticture for the cloud.
+Now that we know about the principles of cloud-native architecture, we now move to discuss a bit more about application architecture for the cloud.
```
Comment (AbhijithGanesh) [inline resources/cloud_native_architecture.md:16]: This is a good catch, I think this patch is good.

File: resources/cloud_native_architecture.md [hunk 6]
```diff
@@ -1,30 +1,32 @@
...
 The following are the 2 popular approaches used by developers to build their applications: <br/><br/>
 **1) Monolithic Architecture**</br>
 **2) Microservices Architecture**
 
-### **Monolithic Architecture**  <hr/>
+### **Monolithic Architecture** <hr/>
+
```
Comment (AbhijithGanesh) [inline resources/cloud_native_architecture.md:22]: Can you remove the <hr> as mentioned above?

File: resources/run-k8ssandra-on-k3d-locally-lightweight/run-k8ssandra-on-k3d-locally-lightweight.md [hunk 7]
```diff
@@ -89,20 +94,22 @@ k3d-my-awesome-cluster-server-2   Ready    control-plane,etcd,master   13m   v1.
 
 ## K8ssandra - lightweight
 
-When following the boilerplate k8ssandra.io installation guide we get a full blown cluster with monitoring and all the bangs and whistles with minimum effort. But this adds on a lot of additional features and complexity that for a simple use case we do not need. To get aquainted with k8ssandra and to have a lightweight develoment installation. The following steps are sufficient.
+When following the boilerplate k8ssandra.io installation guide we get a full blown cluster with monitoring and all the bangs and whistles with minimum effort. But this adds on a lot of additional features and complexity that for a simple use case we do not need. To get acquainted with k8ssandra and to have a lightweight development installation. The following steps are sufficient.
```
Comment (AbhijithGanesh) [inline resources/run-k8ssandra-on-k3d-locally-lightweight/run-k8ssandra-on-k3d-locally-lightweight.md:97]: Good catch!

File: resources/run-k8ssandra-on-k3d-locally-lightweight/run-k8ssandra-on-k3d-locally-lightweight.md [hunk 8]
```diff
@@ -140,28 +150,32 @@ REVISION: 1
...
 
-And we should not have a cassandradatacenters kubernetes object!
+And we should not have a cassandra datacentres kubernetes object!
+
 ```
 kubectl get cassandradatacenters.cassandra.datastax.com -n cass-operator
 ```
 
-Next up it is time to create a minimal example of a cassandra datacenter. For this we take the example-cassdc-minimal.yaml from https://raw.githubusercontent.com/k8ssandra/cass-operator/master/operator/example-cassdc-yaml/cassandra-3.11.x/example-cassdc-minimal.yaml, but we update it with a custom local-path storage class so we can run on k3d using local storage as a storage backend.
+Next up it is time to create a minimal example of a cassandra datacentre. For this we take the example-cassdc-minimal.yaml from https://raw.githubusercontent.com/k8ssandra/cass-operator/master/operator/example-cassdc-yaml/cassandra-3.11.x/example-cassdc-minimal.yaml, but we update it with a custom local-path storage class so we can run on k3d using local storage as a storage backend.
```
Comment (AbhijithGanesh) [inline resources/run-k8ssandra-on-k3d-locally-lightweight/run-k8ssandra-on-k3d-locally-lightweight.md:159]: Please revert it back to `datacenter`

**Review by AbhijithGanesh — CHANGES_REQUESTED** [review]

File: resources/cloud_native_architecture.md (hunk 6 above)
Comment (shefali12-ab) [reply resources/cloud_native_architecture.md:22]: yeah sure

**Comment by shefali12-ab** [comment]
@AbhijithGanesh I have done the changes, it is having resolve the conflict please check what to include.

**Review by AbhijithGanesh — APPROVED** [review]

**Comment by Vrukshali-26** [comment]
LGTM🚀

**Review by Vrukshali-26 — APPROVED** [review]
LGTM🚀

**Review by shreemaan-abhishek — APPROVED** [review]

**Pull request merged**